File: Assignments/C++/Assignment1/helloworld.cpp
//============================================================================
// Name        : helloworld.cpp
// Author      : Soham Halarnkar
// Problem     : 1.1
// Copyright   : Your copyright notice
// Description : Hello World in C++, Ansi-style
//============================================================================

#include <iostream>
using namespace std;

int main() {
	int n;
	cin >> n;
	double arr[n];
	for(int i=0;i<n;i++){
			cin >> arr[i];
		}
	int index = 0;
	double val = 0;
	for(int i=0;i<n;i++){
			if(arr[i]>=45){
				index = i;
				val = arr[i];
				break;
			}

		}
	double minval = INT_MAX;
	double maxval = INT_MIN;
	double average = 0;
	for(int i=0;i<n;i++){
				if(arr[i]<minval){
					minval = arr[i];
				}
				if(arr[i]>maxval){
					maxval = arr[i];
				}
				average += arr[i];
			}
	average = average/n;
	int normal = 0;
	int warning = 0;
	int critical = 0;
	int shutdown = 0;
	for(int i=0;i<n;i++){
	if(arr[i]>=0&&arr[i]<=29){
			normal++;
		}else if(arr[i]>=30&&arr[i]<=44){
			warning++;
		}else if(arr[i]>=45&&arr[i]<=59){
			critical++;
		}else if(arr[i]>=60){
			shutdown++;
		}
	}
	cout << n << " " << "Values entered: ";
	for(int i=0;i<n;i++){
			if(arr[i]<0){
				continue;
			}
			cout << arr[i] << " ";
		}
	cout << "FIRST CRITICAL: " << "INDEX: " << index << " -> " << val << " ";
	cout << "MIN: " << minval << " MAXVAL: " << maxval << " AVERAGE: " << average << " ";
	cout << "NORMAL: " << normal << " WARNING: " << warning << " CRITICAL: " << critical << " SHUTDOWN: " << shutdown << " ";
	return 0;
}

File: Assignments/OOPJ/ForLoop Assignment/helloworld.java

public class helloworld {
	
	public static void main(String[] args) {
		
		//Question 1
		for(int i=1;i<=1000;i++) {
			if(i%2!=0) {
				System.out.println(i);					
			}
		}
		//Question 2
		for(int i=1;i<=500;i++) {
			if(i%2==0) {
				System.out.println(i);					
			}
		}
		//Question 3
				for(int i=1;i<=200;i++) {
					if(i%7==0) {
						System.out.println(i);					
					}
				}
		//Question 4
		for(int i=1;i<=5;i++) {
			for(int j=1;j<=i;j++) {
				System.out.print("*");
			}
			System.out.println();
		}
		
		//Question 5
		for(int i=5;i>0;i--) {
			for(int j=1;j<=i;j++) {
				System.out.print("*");
			}
			System.out.println();
		}
		
		//Question 6
				int l = 1;
				for(int i=1;i<=5;i++) {
					for(int j=1;j<=i;j++) {
						System.out.print(l);
						System.out.print(" ");
						l++;
					}
					System.out.println();
				}
				
		//Question 7
		for(int i=1;i<=5;i++) {
			for(int j=1;j<=i;j++) {
				System.out.print(j);
				System.out.print(" ");
				l++;
			}
			System.out.println();
		}
		
		//Question 8
				for(int i=5;i>0;i--) {
					for(int j=1;j<=i;j++) {
						System.out.print(j);
						System.out.print(" ");
						l++;
					}
					System.out.println();
				}
				
	
	}
}
